webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -16,6 +16,29 @@ interface Props {
 
 const STORAGE_PATH = 'trial-detail';
 
+const chartColorList = [
```

Comment:
**chore:** would you mind putting a one line comment above this explaining that these colors are exactly what plotly uses for the default colors? Just so people don't get the idea to mess with these colors.

Reply:
@hkang1 added the comment. Colors are actually exported in webui/react/node_modules/plotly.js/src/components/color/attributes.js but i was not managing to get them imported (ts typing error, since the plotly file is just plain JS). Do you know if there's a way? Otherwise will merge as is.

Reply:
@apizzini don't worry about trying to import. It's great as-is, thanks! This chart will get a refresh in the near future.